packages/tools/src/tools-routes.tsx

```diff
@@ -1,24 +1,38 @@
-import type { Account } from '@workspace/db/entity/account'
-import type { Network } from '@workspace/db/entity/network'
-import { PortfolioUiNetworkAccountGuard } from '@workspace/portfolio/ui/portfolio-ui-network-guard'
-import { lazy } from 'react'
+import { useDbAccountLive } from '@workspace/db-react/use-db-account-live'
+import { useDbNetworkLive } from '@workspace/db-react/use-db-network-live'
+import { useDbSetting } from '@workspace/db-react/use-db-setting'
+import { lazy, useMemo } from 'react'
 import { useRoutes } from 'react-router'
 
 const ToolsFeatureAirdrop = lazy(() => import('./tools-feature-airdrop.tsx'))
 const ToolsFeatureCreateToken = lazy(() => import('./tools-feature-create-token.tsx'))
 const ToolsFeatureMintToken = lazy(() => import('./tools-feature-mint-token.tsx'))
 const ToolsFeatureOverview = lazy(() => import('./tools-feature-overview.tsx'))
 
-export function Router(props: { account: Account; network: Network }) {
+export default function ToolsRoutes() {
+  const networkLive = useDbNetworkLive()
+  const [activeId] = useDbSetting('activeNetworkId')
+  const network = useMemo(() => networkLive.find((item) => item.id === activeId), [activeId, networkLive])
+  if (!network) {
+    throw new Error('No active network set.')
+  }
```

Comment:
This should probably be in a hook like `useActiveNetwork()`

Reply:
Updated. It's called `useDbNetworkActive` to be consistent with other naming.